### PR TITLE
Move Create PipelineResource to a URL addressable page

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Related to point 5 of https://github.com/tektoncd/dashboard/issues/966 (move create dialogs to full page experience)

Move the Create PipelineResource form to a full page instead
of a modal. This follows the pattern we've already implemented
for Create Secret.

Update the page title.

~~Depends on https://github.com/tektoncd/dashboard/pull/1589~~

**Note:** ignore whitespace for an easier review

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/CreatePipelineResource/CreatePipelineResource.js [hunk 1]
```diff
@@ -64,38 +62,32 @@ export /* istanbul ignore next */ class PipelineResourcesModal extends Component
     };
   }
 
-  componentDidUpdate(prevProps) {
```
Comment (steveodonovan) [inline src/containers/CreatePipelineResource/CreatePipelineResource.js:67]: Are we missing some realtime updates now ?

File: src/containers/CreatePipelineResource/CreatePipelineResource.js [hunk 2]
```diff
@@ -221,11 +213,7 @@ export /* istanbul ignore next */ class PipelineResourcesModal extends Component
     const stateVar = 'type';
     const stateValue = e.selectedItem.text;
     this.setState(prevState => {
-      if (stateValue === 'Git') {
-        gitSource = true;
-      } else {
-        gitSource = false;
-      }
+      gitSource = stateValue === 'Git';
```
Comment (steveodonovan) [inline src/containers/CreatePipelineResource/CreatePipelineResource.js:216]: remove gitsource on line 212 and just define it here ?

File: src/containers/CreatePipelineResource/CreatePipelineResource.js (hunk 1 above)
Comment (AlanGreene) [reply src/containers/CreatePipelineResource/CreatePipelineResource.js:67]: No we're not missing anything, we should never have been requesting ServiceAccounts here as they're not used at all. My best guess is that it was copied along with the structure from another container and never removed.

File: src/containers/CreatePipelineResource/CreatePipelineResource.js (hunk 2 above)
Comment (AlanGreene) [reply src/containers/CreatePipelineResource/CreatePipelineResource.js]: done

**Pull request merged**